python/paddle/fluid/layers/control_flow.py

```diff
@@ -352,7 +350,8 @@ def __init__(self, name=None):
 
     def step(self):
         """
-        The block for user to define operators in RNN.
+        Define operators in each step. step is used in with block, OP in with block
+        will be executed sequence_len times(sequence_len is the length of input)
```

Comment:
add a space between "times" and "(". In English it is written as:
sequence_len times (sequence_len is

Reply:
OK